apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -425,16 +425,44 @@ public async Task WhenSessionPoolReady()
             // End sample
         }
 
-        [Fact]
-        public void GetSessionPoolDatabaseStatistics()
+        [Fact]
+        public void GetSessionPoolStatistics()
         {
             string connectionString = _fixture.ConnectionString;
 
             // Sample: GetSessionPoolDatabaseStatistics
             // Additional: GetSessionPoolDatabaseStatistics
             using (SpannerConnection connection = new SpannerConnection(connectionString))
+            {
+                SessionPool.SessionPoolKeyStatistics stats = connection.GetSessionPoolStatistics();
+                if (stats is null)
+                {
+                    Console.WriteLine("No session pool for this connection string yet");
+                }
+                else
+                {
+                    // Access individual properties...
+                    Console.WriteLine($"Database name: {stats.DatabaseName}");
+                    Console.WriteLine($"Active sessions: {stats.ActiveSessionCount}");
+                    Console.WriteLine($"Pooled read-only sessions: {stats.ReadPoolCount}");
+                    Console.WriteLine($"Pooled read-write sessions: {stats.ReadWritePoolCount}");
+                    // ... or just use the overridden ToString method to log all the statistics in one go:
+                    Console.WriteLine(stats);
+                }
+            }
+            // End sample
+        }
+
+        [Fact]
+        public void GetSessionPoolKeyStatistics()
```

Comment:
We don't need two samples that do the same. Keep the one above with my suggestion on sample name, etc.